modules/ROOT/pages/_partials/sgw-svr-compatibility.adoc

```diff
@@ -181,6 +181,15 @@ Compatibility Matrix
 
 |===
 
+[WARNING]
+====
+Starting from CBS 7.0, the `use_views` feature is no longer available.
+
+* SGW 3.1 will *still* run with `use_views` using a `default scope/default collection` configuration
+* SGW 3.1 will *not* run `use_views` using a `custom scope/custom collection` configuration
```

Comment:
I find this confusing, is it CBS 7.0 or SGW 3.1 that is the issue? 
What are we really trying to tell the user here?

Reply:
Suggestion:
Starting from CBS 7.0, the use_views feature is no longer available.
SGW 3.1 will only run with use_views with a default scope/default collection configuration.
You cannot run use_views with custom scopes and collections.

Reply:
I will take that. Thanks